### PR TITLE
Port forwarding guide and Images for it

### PR DESCRIPTION
**Pull request by ELKER54**

### CODE REVIEW

**Comment by ELKER54** [comment]
I meant to put a comment, this is a big commit. Please DM me on discord for a live preview build for easier reading :)

File: docs/en/server/port-forwarding.md [hunk 1]
```diff
@@ -56,3 +56,189 @@ There are 4 major steps to setting up a port forward.
...
+
+Most routers have a Save button, and some routers require a restart or reboot for the changes to take effect.
+
+### 4. Time to test!
+
+There are a few different ways to test the connection.
+
+Our recommend way is to use a tool called [Probably Up](https://probablyup.net/api) as this tests for BeamMP specific issues and protocols.
+
+This can be done by getting your public IPv4 Address, this once again can be done in a few different ways. The main way it to use a website called [icanhazip.com](https://icanhazip.com/). This is a very simple website which displays your public IP Address. You should be looking for an IP address with the formatting: xxx.xxx.xxx.xxx
```
[the remark this answers is not in the document]
Comment (ELKER54) [reply docs/en/server/port-forwarding.md:230]: Can you please explain what you mean a bit further? Feel free to call me :)

File: docs/en/server/port-forwarding.md [hunk 2]
```diff
@@ -56,3 +56,189 @@ There are 4 major steps to setting up a port forward.
...
+<figure class="image image_resized" style="width:62%;" markdown>
+  ![](../../assets/content/probably-up.png)
+</figure>
+
+Once you have entered in the information required, click on Run test and it should put out something like this:
+
+<figure class="image image_resized" style="width:62%;" markdown>
+  ![](../../assets/content/probably-up-sucessful.png)
+</figure>
+
```
[the remark this answers is not in the document]
Comment (ELKER54) [reply docs/en/server/port-forwarding.md:241]: Forgot about that mb, writing now!

[the remark this answers is not in the document]
Comment (ELKER54) [reply docs/en/server/port-forwarding.md:241]: Done, about to push

**Review by Protogen187 — APPROVED** [review]
@ELKER54 built different 🙏🗿
Approved

**Review by O1LER — APPROVED** [review]

**Comment by ELKER54** [comment]
Gonna close and make new one with fixes, sorry I broke things on my end

**Pull request closed without merging**